code_fellows/stack.py

```diff
@@ -3,20 +3,16 @@
 from code_fellows.linked_list import LinkedList
 
 
-class Stack:
-    def __init__(self, values=None):
-        self.linked_list = LinkedList(values)
-
-    def push(self, value):
-        self.linked_list.push(value)
-        self.head = self.linked_list.head
-
+class Stack(LinkedList):
     def pop(self):
-        self.linked_list.pop()
-
-    def __len__(self):
-        return len(self.linked_list)
+        try:
+            return super().pop().value
+        except ValueError:
+            raise ValueError("Empty Stack")
```

Comment:
This was educational but not a valid solution since the stack has unintended methods inherited from the linked list.